### ISSUE TITLE
von Ravensburger - 

### ISSUE THREAD

**Issue by PaulGuillaumonMuT** [Korion-Interactive/scotland-yard#139]
07.05.24, Valentin:

Buslinien werden als rot und U-Bahn Linien als blau bezeichnet. Es muss aber genau andersherum sein.
Das ist auch in den Regeln noch falsch

**Comment by michaelblatz** [comment]
Das wäre allerdings dann eher eine Frage der Lokalisierung, oder?
Ich nehme an, hier gehts vor allem ums Tutorial und den Regel-Bildschirm.

Für englisch und deutsch könnte ich das anpassen, aber ich weiß nicht, ob sich in anderen Sprachen was an der Grammatik ändert.
Soll ich auf gut Glück selbst anpassen?

**Comment by PaulGuillaumonMuT** [comment]
Das Ticket hab ich erstmal mir zugewiesen, da ich mir das mal anschauen wollte, da es auch noch andere Lokarelevante Meldungen gab.
Sobald ich das im Lokakit soweit fertig habe, melde ich mich wieder.

**Comment by PaulGuillaumonMuT** [comment]
Irgendwie finde ich die Anfrage sehr seltsam.

In den Texten der App sind Buslinien als Rot angegeben.
U-Bahn Linien als Blau.
In der App wird es im Stadtplan dann ja so gewesensein.

Warum ist jetzt in der Portierung die Farbe auf dem Stadtplan für die Linien geändert?
Hängt das mit dem neuen Stadtplan-Design zusammen?
Hat Ravensburger auf dem neuen Stadtplan die Farben von U-Bahn und Bus-Linien getauscht?

**Comment by michaelblatz** [comment]
Das ist korrekt, im neuen Stadtplan sind die Linienfarben anders. 
Alt: Bus ist rot, Bahn ist blau
Neu: Bus ist türkis, Bahn ist rot

**Comment by PaulGuillaumonMuT** [comment]
Also wer sich das bei Ravensburger ausgedacht hat die Farben der Linien aufm Stadtplan auf diese Art und Weise zu ändern.
Das ist doch für Leute, die Scotland Yard schon kennen und dann die neue Version spielen total verwirrend.

Aber egal.
Müssen wir die ganzen Texte anpassen.
Juhuu.

**Comment by michaelblatz** [comment]
Das tut mir Leid.
Ich schätze, das liegt an der dunklen Version von Scotland Yard?
Evtl. wegen passenderer Farbabstimmung?

Ich habe zumindest online keinen dunklen Spieleplan gefunden. Könnte also sein, dass die App hier bereits vorher einen Sonderweg gegangen ist.

**Comment by PaulGuillaumonMuT** [comment]
Stimmt, das kann natürlich sein, dass die App an sich einen Sonderweg gegangen ist und auf den Brettspiel-Plänen war es schon immer anders.

**Comment by michaelblatz** [comment]
Du hast recht, die Haltestellen leuchten noch nicht korrekt. Da sind wir wohl Entwicklungsblind geworden. Das passen wir noch an.

Aber in der Portierung haben Bushaltestellen keine roten Linien, sondern türkise. Nur U-Bahn hat rote, gestrichelte Linien. 
Sollen wir folgendes fixieren?
Bushaltestelle: Türkises Leuchten, analog der Buslinien
U-Bahn: Rotes Leuchten, analog der U-Bahnlinien?

**Comment by PaulGuillaumonMuT** [comment]
Buslinien sind aktuell türkis. Ja. Hatte ich auch nicht geschrieben, dass die aktuell rot sind.
Ich hatte geschrieben, dass das Leuchten um die Bushaltestellen tot ist und eigentlich Blau (Türkis) sein sollte.

Egal.

Ja, legen wir fest
Bushaltestellen blaues (türkises) Leuchten.
U-Bahn: rotes Leuchten

**Comment by PaulGuillaumonMuT** [comment]
Hier das aktuelle Lokafile.
Bitte komplett importieren, da es in allen Sprachen an vielen Stellen Anpassungen gab.
[Loka Scotland Yard.xlsx](https://github.com/Korion-Interactive/scotland-yard/files/15245511/Loka.Scotland.Yard.xlsx)
